### ISSUE TITLE
Reschedule for v111.6.0 release

### ISSUE THREAD

**Issue by tesshucom** [tesshucom/jpsonic#2007]
Considerations for rescheduling the v111.6.0 release.

<details>
<summary>Motivation for rescheduling</summary>

- It's been 3 months since the last release. Jpsonic does not have strict scheduling like business work, but in actuality, version upgrades have been done roughly every three months.
- v111.6.0-SNAPSHOT fixes with scanning, a relatively difficult task. Contains changes that are relatively difficult to separate and many interrelated changes. (It is natural. It is the most important function and affects almost all functions.)
  - The initial schedule is based on rough task groupings. In fact, it will be a few months from now before all of these are resolved.
  - Time itself is not a problem. However, the fixes so far include a number of relatively important bug fixes. (There are also fixes that are difficult to handle with hotfixes)
  - If we continue as originally planned and the release is delayed, the release of bug fixes will be delayed accordingly.

</details>
<details>
<summary>How to divide the remaining tasks</summary>

#### Current remaining tasks - v111.6.0

(This is the order in which the fixes was assumed)

 - #1925
   - #1840 
 - #1927
 - #1445
 - #1373 
 - #1844

##### v111.6.0

It is better not to split the following. Even if it takes some time from now on.

 - #1925
   - #1840 
 - #1927

##### v111.7.0

These require  improvements done in v111.6.0.

 - #1445
 - #1373 
 - #1844

</details>

During the work on v111.6.0, some new proposed changes have arisen. No issues have been raised with it yet.
These should be included in v111.6.0 or v111.7.0 when rescheduled.

 - Remove unnecessary checks from internal diagnostics page. Probably v111.6.0.
   - Originally it was scheduled for : 
     - #1813
   - Simply put, the query has a database incompatibility
   - This check itself does not make much sense in Jpsonic. It is considered more important to correct inconsistencies than to check it.
 - Add a page to view scan logs. Probably v111.7.0.
   - Related to #1928, #1927, etc.
   - Memory tuning will be necessary if parallelization options and video parser switching are implemented in v111.7.0.
     - The question "how much memory should I set" is tricky and probably needs guidance on the wiki.
     - This page plays an important role in v111.7.0, as the scan log also adds the ability to measure memory.

**Comment by tesshucom** [comment]
- Tasks split between v111.6.0 and v111.7.0.
 - #1813 assigned to v111.6.0.
 - #2008 is newly issued and assigned to v111.7.0.
